react-components/src/apps/details/CategoryTree.js

```diff
@@ -1,49 +1,74 @@
 /**
- * @author psarando
+ * @author psarando, sriram
  */
 import React, { Component } from "react";
 import Tree, { TreeNode } from "rc-tree";
 import "rc-tree/assets/index.css";
 
-class CategoryTree extends Component {
-    constructor(props) {
-        super(props);
+import Highlighter from "../../util/Highlighter";
 
-        // This binding is necessary to make `this` work in the callback
-        this.onDetailsCategoryClicked = this.onDetailsCategoryClicked.bind(
-            this
-        );
-    }
+import Book from "../../resources/images/bookIcon.png";
+import Book_Open from "../../resources/images/bookIcon-open.png";
+import Book_Add from "../../resources/images/bookIcon-inbetween.png";
 
-    onDetailsCategoryClicked(selectedKeys, event) {
-        let modelKey = selectedKeys[0];
-        this.props.presenter.onDetailsCategoryClicked(modelKey);
```

Comment:
I missed this in my last review, but with this PR, clicking a category in the App details panel no longer navigates the user to that category in the Apps window.

Are we intentionally removing that functionality? If so, we should also make the new categories un-selectable. Currently you can still select a category, and it highlights that category, but no action is taken.

Reply:
I decided to remove because it doesn't seems to be working in production. But yes I can remove the ability to select category.